### ISSUE TITLE
DAC 23 - Medium Priority WCAG Level AA - Colour Contrast [Non-Text] 10

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#23]
<img width="284" alt="DAC_Colour_Contrast_Non_Text_10" src="https://user-images.githubusercontent.com/75850106/155568246-2e512585-20eb-4adf-9c1e-1a2202240ac5.png">

### Issue
Within the wide range of Moran’s I values diagram, there are multiple colours that are less than the minimum non-text ratio of 3:1. This may make it difficult for low vision users to see the information that the diagram is providing visually.

_Grey_
Foreground: rgb(211, 211, 211) 
Background: #FFFFFF
The contrast ratio is: 1.5:1 
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

_Light blue_
Foreground: rgb(133, 160, 194) 
Background: #FFFFFF
The contrast ratio is: 2.7:1 
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

_Light red_
Foreground: rgb(220, 129, 117) 
Background: #FFFFFF
The contrast ratio is: 2.8:1 
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects

<img width="601" alt="DAC 23 Current code ref" src="https://user-images.githubusercontent.com/75850106/155568503-4ff0ddba-8ab5-453b-995e-f6d13819e54f.png">

### Suggested Solution
Ensure that non-text colours meet a minimum contrast ratio of 3:1 to be visible for low vision users.
